tests/codebuild/create_tests.sh

```diff
@@ -4,28 +4,35 @@ source common.sh
 source inject_tests.sh
 
 # Applies each of the resources needed for the canary tests.
+# Parameter:
+#    $1: Namespace of the CRD
 function run_canary_tests
 {
-  echo "Running canary tests"
+  local crd_namespace="$1"
```

Comment:
We don't want to run ALL the integration and canary tests on both cluster-scoped and namespace-scoped installations? That would effectively double our test time. Perhaps we just need to smoke test the namespacing functionality.

Reply:
These smoke tests would cover all the tests cases we showed off in the demo.

Reply:
Thats a good point. Just the run_canary_tests then? @gautamkmr, do you agree?

Reply:
Well even the canary tests don't check that the namespace scoping is correct. I think it would be nice to see something where we put a trainingjob in another namespace and make sure it never gains a status.

Reply:
Two things 
1. Canary tests should be just subset of tests, which can be representative. For e.g. we don't need to run FSx, EFS test for training job.
2. I think we should have a test for testing this namespace scoped functionality, as Nick mentioned. You don't need to put this test in canary but should be part of integ test

Reply:
I reduced the scope of namespaced tests to one of each job type as in this function `run_canary_tests`. I can reduce this further to a single training job if you want. 

I also added a fail test as Nick mentioned

Reply:
I think that new scope sounds perfect.